source/commands.txt

```diff
@@ -149,8 +149,11 @@ Run these commands from the Command Palette to create and run
        Your playground runs against the deployment specified in your
        active connection.
        
-       |vsce| outputs the results of your playground in the
-       :guilabel:`Output` view in VS Code.
+       |vsce| splits your playground window to display the results in 
+       a pane titled :guilabel:`Playground Results.json` to the right 
+       of your playground. If you disabled split-view for your VS Code 
```

Comment:
Would it be possible to add this same clarification to the other references to split-view?